src/runtime_src/core/edge/common/aie_parser.h

```diff
@@ -28,6 +28,13 @@ class device;
 
 namespace edge { namespace aie {
 
+enum e_tile_type {
```

Comment:
Please use class enum and lower case enumerators, which are scoped.  Let's never again write a plain enum unless we are writing C code :-)